File: 1078-remove-outermost-parentheses/1078-remove-outermost-parentheses.js
/**
 * @param {string} s
 * @return {string}
 */
var removeOuterParentheses = function(s) {
    let res="";
    let count=0
    for (let char of s){
        if(char=='('){
            if(count>0){
                res+=char
            }
            count++;
        }
 
        else if(char===')'){
            count--;
            if(count>0){
                res+=char
            }
        }
    }
    return res
};

File: 1078-remove-outermost-parentheses/1078-remove-outermost-parentheses.py
class Solution(object):
    def removeOuterParentheses(self, s):
        res=[]
        cnt=0
        for char in s :
            if char =='(':
                if cnt>0:
                    res.append(char)
                cnt+=1
            else:
                cnt-=1
                if cnt>0:
                    res.append(char)
        return "".join(res)